src/utils/platform.ts

```diff
@@ -1 +1,2 @@
 export const cloudPlatform = ios ? 'iCloud' : 'Google Drive';
+export const cloudPlatformAccountName = ios ? 'Apple iCloud' : 'Google';
```

Comment:
I think it makes sense to use "Google" here instead of "Google Drive," since then the text will reference their "Google password," but let me know if it should be changed.